library_generation/generate_composed_library.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# This script allows generation of libraries that are composed of more than one
+# service version. It is achieved by calling `generate_library.sh` without
+# postprocessing for all service versions and then calling
+# postprocess_library.sh at the end, once all libraries are ready
+#
+# Arguments
+# --generation_queries: a single string of key-value groups separated by a
+# pipe |. They key-value groups are in the form of `key=value` and will
+# be converted to an argument to generate_library.sh (`--key value`).
+#   example: "key1=value1 key2=value2,,,key1=value3 key2=value4"
+#   In this case, generate_library.sh will be called once with value1 and value2
+#   and once with value3 and value4.
+# --versions_file: list of versions to be applied to the pom.xml and readmes
```

Comment:
When I read "list of versions", I think of "1.0.0, 1.1.0, 1.2.0" as value.

Reply:
I modified this comment to better explain how the `versions.txt` file looks like